app/helpers/service_helper.rb

```diff
@@ -0,0 +1,50 @@
+module ServiceHelper
```

Comment:
helpers are for views. I don't think this is a good place for this code.
Recommend putting it in the service, since it related to that.